crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -1345,7 +1345,7 @@ impl From<&StandardMaterial> for StandardMaterialKey {
 
 impl Material for StandardMaterial {
     fn fragment_shader() -> ShaderRef {
-        PBR_SHADER_HANDLE.into()
+        shader_ref(bevy_asset::embedded_path!("render/pbr.wgsl"))
```

Comment:
This feels like possibly a not so discoverable pattern? I wonder if we should have a clearly-named macro for this particularly and likely common usage?